provider/datagen/src/driver.rs

```diff
@@ -64,12 +45,13 @@ pub enum RuntimeFallbackLocation {
 /// [`Maximal`]: DeduplicationStrategy::Maximal
 /// [`None`]: DeduplicationStrategy::None
 #[non_exhaustive]
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Default)]
 pub enum DeduplicationStrategy {
     /// Removes from the lookup table any locale whose parent maps to the same data.
     Maximal,
     /// Removes from the lookup table any locale whose parent maps to the same data, except if
     /// the parent is `und`.
+    #[default]
     RetainBaseLanguages,
```

Comment:
Issue: `RetainBaseLanguages` is a footgun for blob provider. We've been through this before: blob should default to `Hybrid` (no deduplication).

Reply:
Depending on whether an exporter supports built-in fallback is also a footgun.

Reply:
It's only a footgun if you use deduplication without fallback

Reply:
It's a footgun if you write a custom exporter and forget to override the default impl, it's a footgun when you change exporter and suddenly get different data, it's a footgun, period.

Reply:
> It's a footgun if you write a custom exporter and forget to override the default impl

Not an issue if the trait function is required, which we can do in 2.0

> it's a footgun when you change exporter and suddenly get different data

Not an issue so long as the exporter can handle fallback correctly. It's not a problem to switch from no-deduplication blobs to maximal-deduplication baked with internal fallback.

Reply:
Do you want me to look for all the PRs where this caused issues for me?

Reply:
I simply cannot support a PR that uses deduplication by default in a blob provider because I think that's way worse a footgun than the deduplication strategy changing based on the exporter

I really don't think we need to make deduplication strategy a required parameter, since _we have a reasonable way to get a default_, but if that's the only way to move forward...

Reply:
> Do you want me to look for all the PRs where this caused issues for me?

I recall you having issues with the default impl. We can fix that by making the trait function be required.

I also recall you having issues with trying to migrate from 1.4 options to 1.5 options. I don't see how the default behavior here would positively or negatively impact it.

What else are you thinking of?

Reply:
I don't think inferring strategies from the providers is a reasonable strategy. I'd be very happy with this being a require parameter.

Reply:
How about making this explicit on the API, and inferring it from the `--format` argument in the binary? I'm happy to infer there, because there's a closed set of exporters that we control.

Reply:
^ This seems like an okay middle ground